src/Transformer/StringToDateTimeTransformer.php

```diff
@@ -36,7 +36,13 @@ public function transform(Expr $input, Expr $target, PropertyMapping $propertyMa
         /*
          * Create a \DateTime[Immutable] object from a string.
          *
-         * \DateTimeImmutable::createFromFormat($context[MapperContext::DATETIME_FORMAT] ?? \DateTimeInterface::RFC3339, $input);
+         * ```php
+         * \DateTimeImmutable::createFromFormat(
+         *      $context[MapperContext::DATETIME_FORMAT] ?? \DateTimeInterface::RFC3339,
+         *      $input,
+         *      isset($context[MapperContext::DATETIME_FORCE_TIMEZONE]) ? new \DateTimeZone($context[MapperContext::DATETIME_FORCE_TIMEZONE]) : null
```

Comment:
or I can add a static method `MapperContext::getForcedTimeZone($context)`

this way, we could handle the error if the passed timezone is invalid, and make it explicit that it comes from automapper's context. WDYT?